server/session_test.go

```
@@ -0,0 +1,473 @@
+	session := &sessionTestClient{
+		sessionID:           "blocked-session",
+		notificationChannel: smallBufferChan,
+		initialized:         true,
+	}
+	
+	// Register the session
+	err := server.RegisterSession(context.Background(), session)
+	require.NoError(t, err)
+	
+	// Fill the buffer first to ensure it gets blocked
+	server.SendNotificationToSpecificClient(session.SessionID(), "first-message", nil)
```

golangci-lint (1.64.8)

Error return value of `server.SendNotificationToSpecificClient` is not checked

(errcheck)